Test(SPV-1462) [spv-wallet-go-client][regression-tests] Admin: create test cases for contacts flow

ac4ch opened the pull request:
<!-- 
    Thank you for contributing to our project! Before you submit your Pull Request, please make sure you've completed the items in this checklist.
    Please check the boxes below by putting an x in the [ ] like so: [x]. You can do it right after creating the PR.
-->

# Pull Request Checklist

- [ ] 📖 I created my PR using provided  : [CODE_STANDARDS](https://github.com/bitcoin-sv/spv-wallet-go-client/blob/main/.github/CODE_STANDARDS.md)
- [ ] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/bitcoin-sv/spv-wallet-go-client/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] 🏠 I tested my changes locally.
- [ ] ✅ I have provided tests for my changes.
- [ ] 📝 I have used conventional commits.
- [ ] 📗 I have updated any related documentation.
- [ ] 💾 PR was issued based on the Github or Jira issue.

<!-- 
## PR Title as Conventional Commit

Your PR title should also be a [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/). This will assist us in our release process and in assigning semantic version numbers to releases.
-->

<!-- 
## Related Tickets & Documents

If your changes relate to or close an issue, please include them below. We like to follow [Github's guidance on linking issues to pull requests](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).

For instance, having the text "closes #1234" would link the current pull request to issue number 1234. And when we merge the pull request, Github will automatically close the issue.

- Related Issue #
- Closes #
-->

<!--
## Feature/Issue Details

Your pull request should be tightly connected to a feature or issue. Ensure that you've covered all necessary pathways and there's a possibility to test code behavior following the issue description or feature specification.
-->

wregulski committed:
feat(SPV-1419): manage contacts flow example with proper TOTP generation & validation (#320)

Co-authored-by: chris-4chain <152964795+chris-4chain@users.noreply.github.com>

feat(SPV-1468): change listcontacts package example to be main (#322)

feat(SPV-1469): add linter stage in examples (#323)

Co-authored-by: github-actions <41898282+github-actions[bot]@users.noreply.github.com>

chore(bump): bump golang version (#326)

feat(SPV-1483): add unconfirm contact method for admin API (#325)

build(deps): bump goreleaser/goreleaser-action from 6.1.0 to 6.2.1 in the gh_actions group (#324)

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Michal Gosek <michal.gosek@4chain.studio>

adding users flow

adding users flow

adding users flow

chore: update generated files

troubelshooting contacts

troubelshooting contacts

chore: update generated files

troubelshooting contacts

chore: update generated files

fixing contact tests

chore: update generated files

correcting lint and tag

correcting lint and tag

correcting lint and tag

correcting lint and tag

correcting lint and tag

correcting lint and tag

correcting lint and tag

correcting lint and tag

chore: update generated files

correcting lint and tag

correcting lint and tag

chore: update generated files

correcting lint and tag

chore: update generated files

correcting lint and tag

correcting lint and tag

correcting lint and tag

correcting lint and tag

chore: update generated files

correcting lint and tag

correcting lint and tag

chore: update generated files

correcting lint and tag

github-actions[bot] committed:
chore: update generated files

ac4ch committed:
Merge branch 'main' into test/SPV-1460/User_Create_test_cases_for_contacts_flow

ac4ch committed:
correcting lint and tag

ac4ch committed:
adding access keys

github-actions[bot] committed:
chore: update generated files

ac4ch committed:
correcting imports

github-actions[bot] committed:
chore: update generated files

ac4ch committed:
correcting imports

ac4ch committed:
Merge branch 'test/SPV-1461/User_Create_test_cases_for_access_keys_functionality' of github.com:bitcoin-sv/spv-wallet-go-client into test/SPV-1461/User_Create_test_cases_for_access_keys_functionality

ac4ch committed:
correcting imports

github-actions[bot] committed:
chore: update generated files

ac4ch committed:
correcting tests

github-actions[bot] committed:
chore: update generated files

ac4ch committed:
correcting tests

ac4ch committed:
Merge branch 'test/SPV-1461/User_Create_test_cases_for_access_keys_functionality' of github.com:bitcoin-sv/spv-wallet-go-client into test/SPV-1461/User_Create_test_cases_for_access_keys_functionality

ac4ch committed:
adding admin tests

ac4ch committed:
Merge branch 'test/SPV-1462/Admin_create_test_aases_for_contacts_flow' of github.com:bitcoin-sv/spv-wallet-go-client into test/SPV-1462/Admin_create_test_aases_for_contacts_flow

github-actions[bot] committed:
chore: update generated files

ac4ch committed:
Merge branch 'test/SPV-1461/User_Create_test_cases_for_access_keys_functionality' into test/SPV-1462/Admin_create_test_aases_for_contacts_flow

ac4ch committed:
correcting admin tests

ac4ch committed:
correcting admin tests

github-actions[bot] committed:
chore: update generated files

ac4ch committed:
correcting admin tests

ac4ch committed:
Merge branch 'test/SPV-1462/Admin_create_test_aases_for_contacts_flow' of github.com:bitcoin-sv/spv-wallet-go-client into test/SPV-1462/Admin_create_test_aases_for_contacts_flow

ac4ch committed:
correcting admin tests

ac4ch committed:
correcting admin tests

github-actions[bot] committed:
chore: update generated files

ac4ch committed:
correcting admin tests

ac4ch committed:
Merge branch 'test/SPV-1462/Admin_create_test_aases_for_contacts_flow' of github.com:bitcoin-sv/spv-wallet-go-client into test/SPV-1462/Admin_create_test_aases_for_contacts_flow

mgosek-4chain reviewed (approved):
Minor thing: please add given, when, section to be consistent with another tests across the repository.

ac4ch committed:
correcting admin tests

ac4ch committed:
resolving confilcts

github-actions[bot] committed:
chore: update generated files

ac4ch committed:
resolving confilcts

ac4ch committed:
Merge branch 'test/SPV-1462/Admin_create_test_aases_for_contacts_flow' of github.com:bitcoin-sv/spv-wallet-go-client into test/SPV-1462/Admin_create_test_aases_for_contacts_flow

ac4ch committed:
resolving confilcts

ac4ch committed:
resolving confilcts

ac4ch committed:
correctign tests

github-actions[bot] committed:
chore: update generated files

ac4ch committed:
correctign tests

ac4ch committed:
correctign tests

correctign tests

correctign tests

correctign tests

chore: update generated files

correctign tests

correctign tests

correctign tests

correctign tests

correctign tests
